etc/profile-m-z/make.profile

```diff
@@ -0,0 +1,13 @@
+# Firejail profile for make
```

Comment:
> I'd say that make is much more general-purpose compared to the rest, so I'd be
very cautious in adding such a profile.

None of these profile are in firecfg.config (for reasons) they are provided as an starting point for advanced users who want to sandbox make/meson/... and must be customized (via command-line or .locals) in the most cases. Because build-systems can require anything, you would need to allow anything if you want a general profile.

>  It can be used with any tool and any
programming language, so I don't think include disable-interpreters.inc makes
sense for it.

IMHO adding disable-interpreters by default makes sense, if you need to build something that requires let's say python, then `include allow-python3.inc` in `make.local` or run `firejail --include=/etc/firejail/allow-python3.inc --profile=make make <target>`.

Reply:
Answered here:
<https://github.com/netblue30/firejail/pull/4519#issuecomment-922891679>

Marking as resolved.